Dockerfile

```diff
@@ -68,7 +68,10 @@ ADD tools/docker/supervisord.conf /etc/supervisor/conf.d/nav.conf
 COPY requirements/ /requirements
 ADD requirements.txt /
 ADD tests/requirements.txt /test-requirements.txt
-RUN pip install --upgrade pip && hash -r && pip install -r /requirements.txt && pip install -r /test-requirements.txt
+RUN pip install --upgrade 'pip<10' tox setuptools && \
```

Comment:
I can no longer remember. We did the same in the Docker image for the test environment, because pip 10 introduced "something" that was incompatible with either tox or our build. It may since have been fixed, idk.